contacktbook/DataBase.cs

```diff
@@ -45,17 +52,43 @@ public void updateContact(Contact contact)
                                                               contact.Tel1, contact.Tel2, contact.Email);
                 SQLiteCommand updateCommand = new SQLiteCommand(updateContactTableQuery, connection);
                 updateCommand.ExecuteNonQuery();
+                }
+
+                catch (SQLiteException e)
+                {
+                    LoggerClass logger = new LoggerClass();
+                    logger.writeLog(" SQL Statement Error");
+                    MessageBox.Show("Error!");
+
+                }
+
+                catch (Exception e)
+                {
+                    LoggerClass logger = new LoggerClass();
+                    logger.writeLog(" Unknown error");
+                    MessageBox.Show("Error!");
+                }
             }
         }
-
+ 
         public void deleteContact(Contact contact)

```

Comment:
All the public methods names should start with capital letter. 
Also, they should have documentation comments.